spring-boot-project/spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc

```diff
@@ -56,27 +56,27 @@ defaults. The parent project provides the following features:
 
 * Java 1.8 as the default compiler level.
 * UTF-8 source encoding.
-* A <<using-boot-dependency-management,Dependency Management section>>, allowing you to
-  omit `<version>` tags for common dependencies, inherited from the
+* A <<using-boot-dependency-management,Dependency Management section>> that lets you
+  omit `<version>` tags for common dependencies that are inherited from the
```

Comment:
The meaning's wrong now. It's the dependency management that's inherited rather than the common dependencies

Reply:
I'm not following this sentence. I suspect it needs to be two or perhaps three sentences, but I don't know what they should be. Can you rephrase them?

Reply:
I can try. How about something like this:

A dependency management section, inherited from the `spring-boot-dependencies` pom, that manages the versions of common dependencies. This dependency management allows you to omit `<version>` tags for those dependencies when used in your own pom.

Reply:
That works. Thanks, Andy.